### ISSUE TITLE
[IMPROVEMENT] Scraping using map

### ISSUE THREAD

**Issue by immangat** [CodeForBc/airbnb-regulation#9]
Overview:
-AirBnB ids for Vancouver need to be scraped. Using the map can make the process easier and consistent

Task:
-Define a process to automatically or manually split Vancouver in grids
-Use these grids to search for listing in that area and scrap the ids

**Comment by SamHuo213** [comment]
@immangat , this ticket is to overcome the challenges of 250 limit in listing Airbnbs?

Perhaps you could have a better name for the issue.

**Comment by immangat** [comment]
Update:

-   Done:

    - Use these grids to search for listing in that area and scrap the ids

-   In-Progress

    - Define a process to automatically or manually split Vancouver in grids

**Comment by VimanyuAgg** [comment]
> [@immangat](https://github.com/immangat) , this ticket is to overcome the challenges of 250 limit in listing Airbnbs?
> 
> Perhaps you could have a better name for the issue.

Which limit is this?

**Comment by VimanyuAgg** [comment]
> Update:
> 
> * Done:
>   
>   * Use these grids to search for listing in that area and scrap the ids
> * In-Progress
>   
>   * Define a process to automatically or manually split Vancouver in grids

The [json file](https://github.com/CodeForBc/airbnb-regulation/blob/main/airbnb_project/listings/harvester_app/harvester/spiders/coordinates.json) with co-ordinates - is that the grid we are talking about?

**Comment by immangat** [comment]
Yes. The json is an array of grids. Each grid is defined by 4 coordinates, the four corner vertices of the grid.

In the future, the grids and their coordinates should be dynamic.
